policy/modules/system/userdomain.if

```diff
@@ -3612,6 +3612,44 @@ interface(`userdom_delete_all_user_runtime_named_sockets',`
 	allow $1 user_runtime_content_type:sock_file delete_sock_file_perms;
 ')
 
+########################################
+## <summary>
+##	delete user runtime blk files
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`userdom_delete_all_user_runtime_blk_files',`
+	gen_require(`
+		attribute user_runtime_content_type;
+	')
+
+	allow $1 user_runtime_content_type:dir list_dir_perms;
+	allow $1 user_runtime_content_type:blk_file delete_blk_file_perms;
+')
```

Comment:
why not `delete_blk_files_pattern($1, user_runtime_content_type, user_runtime_content_type)`?

Reply:
I made these interfaces consistent with those already present in `userdomain.if` which are written this way. I don't mind changing them to use the pattern macro though.